lib/project-view.coffee

```diff
@@ -124,8 +138,10 @@ module.exports = ProjectView =
     # Shorten root path if possible
     userHome = fs.getHomeDirectory()
     normRootPath = path.normalize(rootPath)
-    if normRootPath.indexOf(userHome) is 0
-      # Use also tilde in case of Windows as synonym for the home folder
+    if atom.config.get 'project-view.regexMatch'
```

Comment:
Ok makes sense. `/(?:)/` means that it won't match any text so replace will have no effect or? If that is the case then we can skip the `replace` call completely by checking for `if atom.config.get 'project-view.regexMatch' !== ''` which I'd prefer as default behavior.